javaparser-core/src/main/java/com/github/javaparser/ast/expr/StringLiteralExpr.java

```diff
@@ -37,6 +37,7 @@
  * A literal string.
  * <br/><code>"Hello World!"</code>
  * <br/><code>"\"\n"</code>
+ * <br/><code>"\u2122"</code>
  * <br/><code>"™"</code>
  * <br/><code>"💩"</code>
```

Comment:
I am not sure what GitHub wants to tell me with this icon here :)

Reply:
the emoticon :)

Reply:
I knew someone was eventually going to notice ;-)